app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def test_exception_notifications
   protected
 
   def configure_permitted_parameters
-    additional_permissions = [:first_name, :last_name]
+    additional_permissions = [:first_name, :last_name, :photo]
```

Comment:
Why is this in the application controller?

Doesn't this need to be in our version of the RegistrationsController?  (The view associated with this work is the devise/registrations view, so the associated controller and model changes should be related to that.)

Reply:
This is how we add permitted parameters to the devise controller (https://github.com/plataformatec/devise#strong-parameters).  We have not extracted that controller into our local repo as we haven't had the need, and this situation doesn't create that need either.